tasks/ldap.yml

```diff
@@ -2,26 +2,26 @@
 # vim: foldmarker=[[[,]]]:foldmethod=marker
 
 - name: Check default ldap config exists
-  command: php --file "{{ owncloud__home }}/occ" ldap:show-config ""
+  command: '{{ owncloud__occ_bin_file_path | quote }} ldap:show-config'
   changed_when: False
   when: (not ansible_check_mode)
   register: owncloud__register_ldap_default_config_exits
   become_user: '{{ owncloud__user }}'
```

Comment:
In my environment, "Run given occ commands" task didn't work, the output was "Console has to be executed with the user that owns the file config/config.php\nCurrent user: root\nOwner of config.php: www-data\nTry adding 'sudo -u www-data ' to the beginning of the command (without the single quotes)"

So, i have changed "Run given occ commands" task in order to use occ custom command, and then i have changed other tasks with the same problem (LDAP tasks).

And right, i can upload a change to delete become_user, it isn't necessary now.